prime-dotnet-webapi/ViewModels/Enrollee/EnrolleeListViewModel.cs

```diff
@@ -60,5 +60,7 @@ public string CurrentTOAStatus
         public bool AlwaysManual { get; set; }
 
         public bool RemoteAccess { get; set; }
+
+        public ICollection<EnrolleeCareSetting> EnrolleeCareSettings { get; set; }
```

Comment:
This should likely be a flattened collection of CareSettings or care setting IDs rather than EnrolleeCareSettings